### ISSUE TITLE
fetch_orders() parser breaks with quoinex 

### ISSUE THREAD

**Issue by spooky42** [ccxt/ccxt#1876]
Team,
I have the following issue with quoinex and the fetch_orders() method:

```
./fetch-orders.py
Traceback (most recent call last):
  File "./fetch-orders.py", line 17, in <module>
    orders = exchange.fetch_orders()
  File "/usr/local/lib/python3.5/dist-packages/ccxt/qryptos.py", line 344, in fetch_orders
    return self.parse_orders(orders, market, since, limit)
  File "/usr/local/lib/python3.5/dist-packages/ccxt/base/exchange.py", line 1022, in parse_orders
    array = [self.parse_order(order, market) for order in array]
  File "/usr/local/lib/python3.5/dist-packages/ccxt/base/exchange.py", line 1022, in <listcomp>
    array = [self.parse_order(order, market) for order in array]
TypeError: parse_order() takes 2 positional arguments but 3 were given
```

to reproduce, one can use example script (provided)
```
# -*- coding: utf-8 -*-

import os
import sys

root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
sys.path.append(root + '/python')

import ccxt  # noqa: E402

exchange = ccxt.quoinex({
    "apiKey": "#######",
    "secret": "########",
    "enableRateLimit": True,
})

orders = exchange.fetch_orders()  # raises
print(orders)

order = exchange.fetch_order(orders[0]['id'])
print(order)
```

- OS: Ubuntu 16.04
- Programming Language version: python 3.5
- CCXT version: ccxt-1.10.1076-py2.py3-none-any.whl
- Exchange: quoinex
- Method: fetch_orders

Am I doing something wrong?

**Comment by kroitor** [comment]
Fixed in 1.10.1080. Thanks for reporting!

**Comment by spooky42** [comment]
Igor,

Thanks for prompt response! Just upgraded to ccxt-1.10.1080 and tested it out. It still breaks somewhere in the parser:

```
./fetch-orders.py
Traceback (most recent call last):
  File "fetch-orders.py", line 17, in <module>
    orders = exchange.fetch_orders()
  File "/usr/local/lib/python3.5/dist-packages/ccxt/qryptos.py", line 346, in fetch_orders
    return self.parse_orders(orders, market, since, limit)
  File "/usr/local/lib/python3.5/dist-packages/ccxt/base/exchange.py", line 1022, in parse_orders
    array = [self.parse_order(order, market) for order in array]
  File "/usr/local/lib/python3.5/dist-packages/ccxt/base/exchange.py", line 1022, in <listcomp>
    array = [self.parse_order(order, market) for order in array]
  File "/usr/local/lib/python3.5/dist-packages/ccxt/qryptos.py", line 286, in parse_order
    market = self.markets_by_id[market]
KeyError: None
```

**Comment by kroitor** [comment]
@spooky42 my bad, left a typo there, uploaded a refix, will be available in 5-10 minutes in v1.10.1081.

**Comment by spooky42** [comment]
@kroitor 

Yep, that works. Kudos.

```
python3 ./fetch-orders.py
[{'type': 'stop', 'remaining': 0.0, 'amount':
...
'symbol': 'BTC/USD'}]
{'type': 'stop', 
...
'trades': None, 'symbol': 'BTC/USD'}
```
